docs/release-notes/observability.st

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**Improvements**
+
+Kubernetes: Add Determined resource information such as `workspace` and `task ID` as pod labels. This improvement facilitates better resource tracking and management within Kubernetes environments.
+
+Configuration: Introduce a DCGM Helm chart and Prometheus configuration to the `tools/observability` directory. Additionally, two new dashboards, "API Monitoring" and "Resource Utilization", have been added to improve observability and operational insight.
```

Comment:
this is an "st" file, please change it to an *.rst file

"docs/release-notes/observability.st" should be "docs/release-notes/observability.rst"

Reply:
Good catch - thank you!